src/app/index.html

```diff
@@ -2,11 +2,13 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link href="./app.css" rel="stylesheet"/>
     <title>Hello World + React</title>
   </head>
   <body>
-    <div id="hello"></div>
-    <div id="world"></div>
+    <!--div id="hello"></div-->
```

Comment:
@yshen01 Think we can get rid of these and their tests and replace with tests for your component